### PR TITLE
Update test 01 to add content check 

### PR DESCRIPTION
**Pull request by bdematte**
Updating the test case 01 to include the comparison of the sync'ed content

### CODE REVIEW

**Comment by bdematte** [comment]
This has a dependency on https://github.com/openshift/boilerplate/pull/15

File: boilerplate/test/test_base_convention/.codecov.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+codecov:
```
Comment (2uasimojo) [inline boilerplate/test/test_base_convention/.codecov.yml:1]: Still don't want the files in test conventions to be real files.

File: test/case/01_bootstrap_update [hunk 2]
```diff
@@ -18,4 +18,16 @@ cd $repo
...
+check_update 
+
+if [ $? -ne 0 ] ; then
+	exit $?
+fi
+
+add_convention test/test_base_convention
+make update_boilerplate
+
+check_update
```
Comment (2uasimojo) [inline test/case/01_bootstrap_update:31]: I tried to get this to fail by adding a line here that modifies one of the convention files before `check_update`. It didn't work. I didn't dig in.

File: boilerplate/test/test_base_convention/.codecov.yml (hunk 1 above)
Comment (bdematte) [reply boilerplate/test/test_base_convention/.codecov.yml:1]: Even if they don't have to be real life ones, is there any specific issue you are thinking of for having to differ ?

Comment (2uasimojo) [reply boilerplate/test/test_base_convention/.codecov.yml:1]: Sorry, I'm not quite parsing your question.

I'm just complaining that it's confusing to see real content in these files. For example, somebody in the future is going to think they need to keep them in sync with what's in the real conventions.

It's really not a big deal though.

**Comment by 2uasimojo** [comment]
I ran this through a few times, injecting various failures, and verified it works as expected. Let's merge this now and worry about making the content of the test convention less meaningful in a followup PR.

/lgtm
/approve

**Pull request merged**